packages/containerd-1.7/containerd-config-toml_k8s_containerd_sock

```diff
@@ -75,3 +81,10 @@ identitytoken = "{{{identitytoken}}}"
 {{/if}}
 {{/each}}
 {{/if}}
+
+# Plug soci snapshotter into containerd
+[proxy_plugins.soci]
+type = "snapshot"
+address = "/run/soci-snapshotter/soci-snapshotter.sock"
+[proxy_plugins.soci.exports]
+  root = "/var/lib/soci-snapshotter"
```

Comment:
I don't think the extra space at the beginning is needed. We certainly don't add them for other sections in the config file.

Reply:
You're correct it is not necessary, indenting is this style is usually done as a visual aid to show parent-child settings. I'll remove the space